taccsite_cms/static/site_cms/css/src/_imports/components/django.cms.blog.app.css

```diff
@@ -88,18 +83,16 @@ Styleguide Components.DjangoCMS.Blog.App
 /* Metadata */
 
 /* To layout data */
-.app-blog .attrs,
 .app-blog .tags,
 .app-blog .categories {
   display: flex;
   flex-direction: row;
   gap: 1em;
 }
-.app-blog .attrs {
-  justify-content: space-between;
-}
-.app-blog .date {
-  order: -1
+
+/* To always hide byline prefix */
+.app-blog .byline > span {
+  display: none;
 }
```

Comment:
**Why even have `trans "by"` if you're not gonna use it?** So another site can choose to show it, designers can change their mind, and I can continue to match the template I cloned and overwrite from the blog app[^1].

[^1]: Until I finally give in and make it my own, because the default template is not gospel, Wes; it's just an example.